### PR TITLE
refactor: move release code to a package (@miodec)

### PR DESCRIPTION
**Pull request by Miodec**
Moves all release related things to @monkeytype/release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
     "start-fe": "turbo run start --filter @monkeytype/frontend",
     "docker": "cd backend && docker compose up",
     "audit-fe": "cd frontend && npm run audit",
-    "release": "node ./bin/release.mjs",
-    "release-fe": "node ./bin/release.mjs --fe",
-    "release-be": "node ./bin/release.mjs --be",
-    "release-no-deploy": "node ./bin/release.mjs --no-deploy",
-    "release-dry": "node ./bin/release.mjs --dry",
+    "release": "monkeytype-release",
```
Comment (fehmer) [inline package.json:32]: don't we need `build` first?

File: packages/release/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@monkeytype/release",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "nodemon --watch src --exec 'npm run release-dry'",
+    "dev-changelog": "nodemon ./src/buildChangelog.js",
+    "release": "node ./src/index.js",
```
Comment (fehmer) [inline packages/release/package.json:8]: unused? you call them from the root level?

File: package.json (hunk 1 above)
Comment (Miodec) [reply package.json:32]: handled by the release script (all though i just realised the path will be wrong)

File: packages/release/package.json (hunk 2 above)
Comment (Miodec) [reply packages/release/package.json:8]: removing

**Review by fehmer — APPROVED** [review]
👍

**Pull request merged**